src/Nox.Types/Types/Color/Color.cs

```diff
@@ -59,10 +60,21 @@ public static Color From(byte alpha, byte red, byte green, byte blue)
     /// <param name="color">The origin value to create the <see cref="Color"/> with</param>
     /// <returns></returns>
     /// <exception cref="TypeValidationException"></exception>
-    public static Color From(string color)
+    public static new Color From(string colorHex)
```

Comment:
I don't think we need this From method, as it is already part of the base ValueObject.

Reply:
done